index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr Norman Borlaug</h1>
+      <p id="p">The man who saved a billion lives</p>
+      <figure id="img-div">
+        <img id="image" src="/Images/tribute-page-main-image.jpg" alt="image" />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </figcaption>
+      </figure>
+       <div id="tribute-info">
```

Comment:
Please add <section> element instead of <div> to  have more  semantic meaning.

Reply:
> Please add
> 
> element instead of
> to have more semantic meaning.

- [ ] Good one giving the WHY but don't you think these are three incomplete sentences, kindly have your comment on a line and it should break when a line is full. Some things like "Please add element instead of... to have more semantic meaning."
- [ ] Also, kindly complete the instead for it not to be nebulous.